test/snapshots/blueprint.test.ts.md

```diff
@@ -848,6 +848,8 @@ Generated by [AVA](https://avajs.dev).
               title: 'List foos',
             },
           ],
+          isDeprecated: false,
+          isUndocumented: false,
```

Comment:
We need to add an example to the test fixture for a route and namespace where both are true.